cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java

```diff
@@ -818,6 +851,18 @@ Future<ReconciliationState> reconcileEntityOperator(Clock clock)    {
                     .reconcile(pfa.isOpenshift(), imagePullPolicy, imagePullSecrets, clock)
                     .map(this);
         }
+
+        /**
+         * @return true if the autorebalance is enabled within the Cruise Control configuration, false otherwise
+         */
+        boolean isAutoRebalancingEnabled() {
+            // Cruise Control is not defined in the Kafka custom resource or no autorebalance enabled, so nothing to reconcile
+            if (kafkaAssembly.getSpec().getCruiseControl() == null || kafkaAssembly.getSpec().getCruiseControl().getAutoRebalance() == null) {
+                LOGGER.debugCr(reconciliation, "Cruise Control or inner autorebalance field not defined in the Kafka custom resource, no auto-rebalancing to reconcile");
```

Comment:
Do you think we really need this message at DEBUG level? I would probably delete it ... but if you think it is useful, feel free to reject this comment.

Reply:
Tbh I would leave it. If you prefer we can move it to trace?